autoprotocol/protocol.py

```diff
@@ -4920,12 +4920,6 @@ def provision(self, resource_id, dests, volumes):
         ValueError
             Volume to provision exceeds max capacity of well
 
-        Returns
-        -------
-        Provision
-            Returns the :py:class:`autoprotocol.instruction.Provision`
-            instruction created from the specified parameters
-
```

Comment:
hmm.. I think we still want to ensure we return for this Instruction, that is something we do for all Instruction-creation methods. The intent there is that the user can get a handle on the Instruction and eventually feed it into a time constraint or similar construct.

I think pylint caught a bug here where it inspected the output and realized that we were not returning in all cases.

Reply:
Made changes to return only `Provision` instructions.

BTW, in the previous commit I removed return type even from documentation. Not sure how my local `pre-commit` hook missed it